reactive/kotlinx-coroutines-rx2/build.gradle

```diff
@@ -9,10 +10,12 @@ dependencies {
     api "io.reactivex.rxjava2:rxjava:$rxjava2_version"
 }
 
-tasks.withType(dokka.getClass()) {
-    externalDocumentationLink {
-        url = new URL('http://reactivex.io/RxJava/2.x/javadoc/')
-        packageListUrl = projectDir.toPath().resolve("package.list").toUri().toURL()
+tasks.withType(DokkaTaskPartial.class) {
+    dokkaSourceSets.configureEach {
+        externalDocumentationLink {
+            url.set(new URL('http://reactivex.io/RxJava/2.x/javadoc/'))
+            packageListUrl.set(projectDir.toPath().resolve("package.list").toUri().toURL())
+        }
```

Comment:
Looking through `git grep externalDoc`, I notice the lack of Swing and JavaFX modules. It looks like our documentation doesn't reference anything Swing-related, so such a directive is not needed there, but the same is not true for JavaFX.

Also, why does this look different from the way it's done in the [reactive module](https://github.com/Kotlin/kotlinx.coroutines/blob/b852b20131fff5c0d605d7cb5f8001b715f02c1e/reactive/kotlinx-coroutines-reactive/build.gradle.kts#L34-L36)? It looks much more succinct there, and I don't understand the difference between these two modules in this regard.

Reply:
FTR: discussed that reactive module leverages our kts extensions

Regarding JavaFX -- seems to be a general (not PR-related oversight) that should be fixed